cmd/tsbs_generate_data/main.go

```diff
@@ -226,7 +235,7 @@ func runSimulator(sim common.Simulator, serializer serialize.PointSerializer, ou
 		if currGroup == groupID {
 			err := serializer.Serialize(point, out)
 			if err != nil {
-				fatal("%v", err)
+				fatal("can not serialize point to out: %v", err)
```

Comment:
`cannot serialize point: %s`

(A user won't necessarily know/care the var names we use and it will tend to get out of sync if those things change)

Reply:
Yes, I agree